pkg/server/http_status.go

```diff
@@ -733,3 +739,118 @@ func (s *Server) newStatsPriorityQueueHandler() *optimizor.StatsPriorityQueueHan
 
 	return optimizor.NewStatsPriorityQueueHandler(do)
 }
+
+// Event defines the event structure for tracing.
+// See https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/preview?tab=t.0
+type Event struct {
```

Comment:
The trace event format related code is better placed in the traceevent package than in the http module.